packages/office-ui-fabric-react/src/components/Dropdown/examples/Dropdown.Basic.Example.tsx

```diff
@@ -42,6 +43,7 @@ export class DropdownBasicExample extends React.Component<any, any> {
         />
 
         <Dropdown
+          className="ms-Dropdown-example"
```

Comment:
Not something we need to address here, but I'm going to clean up the examples so that we only use the `ms-` prefix internally in the components, and not in our examples. I see developers adding `ms-` to product code all over the place, when it really should indicate "this is a Fabric class".